packages/vapor/scss/controls/radios.scss

```diff
@@ -68,12 +68,9 @@
         }
 
         &:checked + label {
-            padding-top: $radio-button-checked-text-padding-top;
-            font-weight: var(--main-font-bolder);
-
             &:before {
                 z-index: 1;
-                padding: 5px;
+                padding: 7px 5px 5px 5px;
```

Comment:
![image](https://user-images.githubusercontent.com/22773767/110852872-e38c9c80-8280-11eb-97f2-f1c1ee989b15.png)

I know its is a SUPER tiny point but I think 7px is a smidge too low down, 6px puts in in the middle. But everything else is superb

Reply:
:eagle: Nice catch!